addons/l10n_cz/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Czech - Accounting',
+    'version': '1.0',
+    'author': '26HOUSE',
+    'website': 'http://www.26house.com',
+    'category': 'Accounting/Localizations/Account Charts',
+    'description': """
+Czech accounting chart and localization.
+
+
+Tento modul definuje:
+
+- Českou účetní osnovu za rok 2020
+
+- Základní sazby pro DPH z prodeje a nákupu
+
+- Základní fiskální pozice pro českou legislativu
+    """,
+    'depends': [
+        'account',
+        'base_iban',
+        'base_vat',
+        'l10n_multilang',
```

Comment:
Why do we need multilang here?  The main language for the CoA seems to be Czech already.  The same could be said about base_iban/base_vat: I don't see an immediate dependency, although it can be in handy to have it by default.